packages/viewer/src/lib/scenes/scene.ts

```diff
@@ -259,6 +259,28 @@ export class Scene {
     );
   }
 
+  /**
+   * Applies the specified features of the provided scene view state to the scene.
+   */
+  public async applyPartialSceneViewState(
+    sceneViewStateId: UUID.UUID,
+    featuresToApply: string[],
```

Comment:
I don't think that we should accept any payload from the client - we should create a new feature type that is public facing to expose to developers which features are accessible to partially apply.